packages/blade/src/components/Counter/Counter.tsx

```diff
@@ -47,24 +47,36 @@ type ColorProps = {
   backgroundColor: StyledCounterProps['backgroundColor'];
 };
 
+const isFeedbackVariant = (variant: string): variant is Feedback => {
+  const feedbackVariants = ['information', 'negative', 'neutral', 'notice', 'positive'];
+  return feedbackVariants.includes(variant);
+};
+
 const getColorProps = ({
-  intent = 'neutral',
+  variant = 'neutral',
   contrast = 'low',
 }: {
-  intent: NonNullable<CounterProps['intent']>;
+  variant: NonNullable<CounterProps['variant']>;
   contrast: NonNullable<CounterProps['contrast']>;
 }): ColorProps => {
   const props: ColorProps = {
-    textColor: `feedback.text.${intent}.${contrast}Contrast`,
-    backgroundColor: `feedback.background.${intent}.${contrast}Contrast`,
+    textColor: 'feedback.text.neutral.lowContrast',
+    backgroundColor: 'feedback.background.neutral.lowContrast',
   };
+  if (isFeedbackVariant(variant)) {
+    props.textColor = `feedback.text.${variant}.${contrast}Contrast`;
+    props.backgroundColor = `feedback.background.${variant}.${contrast}Contrast`;
+  } else {
+    props.textColor = `badge.text.${variant}.${contrast}Contrast`;
```

Comment:
I've used badge's tokens only since we want Badge and Counter to be related. Any changes to badge should also reflect it on counter.

Reply:
can we rather abstract this out as `feedback`? so all of the counters, badges, alerts, toasts(in future) etc can use the common thing?

Reply:
You mean adding `blue` to `feedback`? That was intentionally not done because we don't want every feedback component to support blue